src/WebPages/signup/Signup.jsx

```diff
@@ -0,0 +1,139 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import { useAuth } from "../../Contexts/auth-context";
+import { useNavigate } from "react-router-dom";
+import "./Signup.css";
+
+const Signup = () => {
+  const navigate = useNavigate();
+  const { signupHandler, token } = useAuth();
+  const [signUpDetails, setSignUpDetails] = useState({
+    email: "",
+    password: "",
+    firstname: "",
+    lastname: "",
+    confirmpassword: "",
+  });
+
+  useEffect(() => {
+    if (token) {
+      setTimeout(() => {
+        navigate("/");
+      }, 1000);
+    }
+  }, [token]);
+
+  return (
+    <>
+      <div className="signup-page">
+        <div className="login-container">
+          <div className="login-inputs">
+            <h1>Sign Up</h1>
+            <div className="input">
+              <label>Email</label>
+              <input
```

Comment:
Your label and input element are not connected. There is no point in using labels without htmlFor attribute.
Why you should do this? 
It increases the tap target.

Reply:
Thanks for your review manish . I'll do the changes